SWENT-GRP5/SOLVIT
# feat: improve notifications

## Changes

- Add provider name to request acceptance notification for better user experience
- Set SOLVIT logo as notification icon for consistent branding
- Fix provider data fetching in PendingJobsSection

## Testing

1. Log in as a provider
2. Accept a pending request
3. Verify that the notification shows the provider's name
4. Verify that the notification shows the SOLVIT logo

Fixes #480
